index.html

```diff
@@ -65,6 +65,37 @@
     <!-- end of Sponsors -->
 
     <!-- FOOTER -->
+    <footer class="container">
+      <div class="row">
+        <div class="col-12 col-sm-6 col-lg-5">
+          <h4>About Me</h4>
+          <p>
+            We have tested a number of registry fix and clean utilities and
+            present our top 3 list on our site for your convenience
+          </p>
+          <p>
+            Copyright ©2022 All rights reserved | This template is made with
+            &hearts; by <a href="#">Colorlib</a>
+          </p>
+        </div>
+        <div class="col-12 col-sm-6 col-lg-5">
+          <h4>Newsletter</h4>
+          <p>Stay updated with our latest trends</p>
+          <form class="col-12 col-md-6 ml-lg-1">
+            <input type="email" placeholder="Enter Email Address" />
```

Comment:
Nepavyko input lauko padaryti didesnio? Bandei ar reaguoja i musu col dydzius?

Reply:
Tai as input tik del bendro vaizdo idejau be stiliaus kol kas kad nereiktu tik zodzio Form rasyt.

Reply:
ok